src/app/styles.tsx

```diff
@@ -0,0 +1,145 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+
+export const PageContainer = styled.div`
+  width: 100%;
+  min-height: 100vh;
+`;
+
+export const TitleSection = styled.div`
+  position: relative;
+  width: 100%;
+  min-height: 100vh; // Use 100vh to make the div fill the entire viewport height
+  display: flex;
+  flex-direction: column;
+  overflow-y: hidden;
+`;
+
+export const ButtonDiv = styled.div`
+  margin-top: 25px;
+  width: clamp(224px, 25%, 500px);
+  height: clamp(40px, 5vh, 60px);
+`;
```

Comment:
Upon inspection, it seems that the button has a font size of `16px` instead of the `24px` specified on Figma. Maybe we can add `font-size: 1.5rem !important` here to force it to `24px`